main/ZgatewayBT.ino

```diff
@@ -471,9 +608,12 @@ void coreTask(void* pvParameters) {
         if (low_power_mode == 2)
           digitalWrite(LOW_POWER_LED, 1 - LOW_POWER_LED_OFF);
         BLEscan();
-        launch_discovery();
+        // Launching a connect every BLEscanBeforeConnect
+        if (!(scanCount % BLEscanBeforeConnect) || scanCount == 1)
```

Comment:
```suggestion
        if (scanCount % BLEscanBeforeConnect == 0)
```

Reply:
We need to do the BLEconnect after the first scan if we change it like that we will only do it at `scanCount = BLEscanBeforeConnect`
This is required by the low power mode, this mode puts the ESP to sleep after the BLEscan and the BLEconnect operations.